src/plugin/vk-wrapper/graphicsPipeline/GraphicsPipeline.hpp

```diff
@@ -0,0 +1,77 @@
+/**************************************************************************
+ * VkWrapper v0.0.0
+ *
+ * VkWrapper is a software package, part of the Engine².
+ *
+ * This file is part of the VkWrapper project that is under GPL-3.0 License.
+ * Copyright © 2024 by @MasterLaplace, All rights reserved.
+ *
+ * VkWrapper is a free software: you can redistribute it and/or modify
+ * it under the terms of the GPL-3.0 License as published by the
+ * Free Software Foundation. See the GPL-3.0 License for more details.
+ *
+ * @file GraphicsPipeline.hpp
+ * @brief GraphicsPipeline class declaration.
+ *
+ *
+ * @author @MasterLaplace
```

Comment:
Here you are allowed to put your name only though, as you are the only one that contributed to this file. (just resolve this comment when you see it)